sktime/transformations/series/multivariate_compositor.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__all__ = ["MultivariateCompositor"]
+__author__ = ["Svea Meyer"]
+
+import pandas as pd
+from sklearn.base import clone
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.utils.validation.series import check_series
+
+
+class MultivariateCompositor(_SeriesToSeriesTransformer):
```

Comment:
why not to name this class `ColumnTransformer` as in `sklearn`? It would be a name conflict maybe if s.o. wants to use both together. So what do you think about `ColumnComposition`. I think the term "multivariate" is in general weakly defined and often confusing if you talk about endog or exog multivariate, thats why I would propose to remove the name "multivariate" from the class name. What do you think?

Reply:
yes, for sure - I don't mind renaming it

Reply:
ok, but I am not sure which name would be best. Any ideas @mloning @fkiraly ?